packages/docs/SUMMARY.md

```diff
@@ -19,7 +19,7 @@
 - [Validator Setup FAQ](getting-started/validator-troubleshooting-faq.md)
 - [Glossary](GLOSSARY.md)
 
-## Celo Gold Holder Guide
+## CELO Holder Guide
```

Comment:
This is going to change the URL. We need redirects for the old URLs.

Reply:
changed the links in commit `9598f9cc8df7103408ba1d38e8dda28359c711c9`

Reply:
But how do we add redirects?

Reply:
addressed!